CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Optimizely Objective-C SDK Changelog
 
+## 3.1.0
+May 15th, 2019
+
+## New Features:
+* Introduced Decision notification listener to be able to record:
+    * Variation assignments for users activated in an experiment.
+    * Feature access for users.
+    * Feature variable value for users.
+    * Added APIs to be able to conveniently add Decision notification handler (addDecisionNotificationHandler) and Track notification handler (addTrackNotificationHandler).
+
+## Bug Fixes:
+* Feature variable APIs return default variable value when featureEnabled property is false. (#390)
+
+## Deprecated
+* Activate notification listener is deprecated as of this release. Recommendation is to use the new Decision notification listener. Activate notification listener will be removed in the next major release.
+* addActivateNotificationListener, addTrackNotificationListener and addNotificationListener APIs on NotificationCenter.
```

Comment:
I don't think we refactored code in this SDK. Please take a look.

Reply:
We added a deprecation notice in the header: https://github.com/optimizely/objective-c-sdk/blob/master/OptimizelySDKCore/OptimizelySDKCore/OPTLYNotificationCenter.h#L101

Reply:
You are right, we only added a deprecation notice in the `addActivateNotificationListener` method: https://github.com/optimizely/objective-c-sdk/blob/master/OptimizelySDKCore/OptimizelySDKCore/OPTLYNotificationCenter.h#L101